File: apps/mobile/app/lib/features/trip/domain/trip_status.dart
/// Совпадает с `TripStatus` на бэкенде (числовой enum).
abstract final class TripStatusCode {
  static const int pending = 0;
  static const int started = 1;
  static const int active = 2;
  static const int paused = 3;
  static const int finished = 4;
  static const int cancelled = 5;
  static const int error = 6;
  static const int unknown = 7;
}

File: apps/backend/src/modules/trip/entities/trip.status.ts
export enum TripStatus {
  PENDING = 0,
  STARTED = 1,
  ACTIVE = 2,
  PAUSED = 3,
  FINISHED = 4,
  CANCELLED = 5,
  ERROR = 6,
  UNKNOWN = 7,
}

/** Статусы незавершённой поездки (H11, partial unique index в БД). */
export const ONGOING_TRIP_STATUSES: TripStatus[] = [
  TripStatus.PENDING,
  TripStatus.STARTED,
  TripStatus.ACTIVE,
  TripStatus.PAUSED,
];
